contrib/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ add_contrib (protobuf-cmake protobuf)
 add_contrib (openldap-cmake openldap)
 add_contrib (grpc-cmake grpc)
 add_contrib (msgpack-c-cmake msgpack-c)
+add_contrib (BLAKE3)
```

Comment:
It can be useful to keep the possibility to build ClickHouse without Rust and its components.